src/schema/sale_artwork.js

```diff
@@ -24,6 +25,51 @@ export const isBiddable = (sale, { artwork: { sold } }) => {
   return !sold && sale.is_auction && sale.auction_state === "open"
 }
 
+const bid_increments_calculator = ({
```

Comment:
Out of curiosity, is MP set up to be able to use `async` functions? That would make methods like this a bit more straightforward.

Reply:
Mmm it is! Good idea.